src/tactic/refine_struct.lean

```diff
@@ -0,0 +1,50 @@
+import tactic.interactive
```

Comment:
move to `test/refine_struct.lean`?

Reply:
Oops, thanks.

Reply:
Oh, there's a merge conflict now with #2317.